src/Gui/SoDatumLabel.cpp

```diff
@@ -1535,7 +1548,7 @@ void SoDatumLabel::drawArcLength(const SbVec3f* points, float& angle, SbVec3f& t
     float startangle = atan2f(vc1[1], vc1[0]);
     float endangle = atan2f(vc2[1], vc2[0]);
     if (endangle < startangle) {
-        endangle += 2.0F * (float)M_PI;
+        endangle += 2.0F * (float)pi;
```

Comment:
Really should be `std::numbers::pi_v<float>` and no cast (can be later work if you want).

Reply:
I was thinking that it could be future work